cypress/e2e/models/migration/migration-waves/migration-wave.ts

```diff
@@ -276,4 +276,38 @@ export class MigrationWave {
         cy.contains("li", date.toLocaleString("en-us", { month: "long" })).click();
         cy.contains("button:not([disabled])", `${date.getDate()}`).first().click();
     }
+
+    public clickWaveStatus() {
+        cy.contains(this.name)
+            .parents("tr")
+            .within(() => {
+                cy.get(MigrationWaveView.waveStatusColumn).click();
+            });
+    }
+
+    public isExpanded() {
+        cy.contains(this.name)
+            .parent()
+            .parent()
+            .should(($element) => {
+                expect(
+                    $element,
+                    `Migration Wave with name ${this.name} is not expanded`
+                ).to.have.class(MigrationWaveView.waveExpanded);
+            });
+    }
+
+    public removeApplication(applicationName) {
```

Comment:
If an application is removed from the migration wave in the UI it has to be also removed from the `applications` property of the model.